systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java

```diff
@@ -481,10 +489,11 @@ public String getDeploymentSnippet() throws InterruptedException {
         StringBuilder addressSpaceDeployment = new StringBuilder();
         for (int i = 0; i < RETRY_COUNTER && addressSpaceDeployment.toString().isEmpty(); i++) {
             List<WebElement> snippetElements = selenium.getDriver().findElements(By.xpath("//div[@class='ace_line']"));
-
             for (WebElement currentElement : snippetElements) {
-                if (currentElement.getText().contains("kubectl")
-                        || currentElement.getText().isEmpty() || currentElement.getText().equals("EOF")) {
+                if (!currentElement.getText().contains("oc")) {
```

Comment:
`.contains("oc")` seems a bit weak.   It could match spuriously elsewhere in the command.  Would the following work?
```suggestion
                if (!currentElement.getText().startsWith("oc ")) {
```